estate/views/estate_menus.xml

```diff
@@ -0,0 +1,7 @@
+<odoo>
+    <menuitem id="estate_menu_root" name="Real Estate">
+        <menuitem id="estate_first_level_menu" name="Advertisements">
```

Comment:
We also have convention for xml ids, although they are quite recent and therefore not that often enforced in the code for the moment.
I suggest you read this section again and update them accordingly :slightly_smiling_face: 
https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#xml-ids-and-naming